docs/source/features/sheet_management/dataframe.rst

```diff
@@ -96,6 +96,36 @@ Read a DataFrame cell:
  Edit the DataFrame
 ********************
 
+Edit Rows
+=========
+
+Edit one row.
+
+.. code:: python
+
+   df.loc[0] = {"filename": "d.jpg"}
+
+Edit multiple rows.
+
+.. code:: python
+
+   df.loc[0:1] = [{"filename": "d.jpg"}, {"filename": "e.jpg"}]
+
+Edit The Items Of Column
+========================
+
+Edit one item.
+
+.. code:: python
+
+   df["filename"][0] = "d.jpg"
+
+Edit multiple items.
+
+.. code:: python
+
+   df["filename"][0:1] = ["d.jpg", "e.jpg"]
```

Comment:
do we need to add some examples like File, DataFrame?

Reply:
we introduce them in thier own section.